### PR TITLE
Unfork activation email

### PR DESCRIPTION
**Pull request by stvstnfrd**
but keep the link to ZenDesk.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
retest this please

**Comment by stvstnfrd** [comment]
retest this please

File: lms/templates/emails/activation_email.txt [hunk 1]
```diff
@@ -16,9 +16,8 @@ ${_("There's just one more step before you can enroll in a course: "
...
-
-${_("If you didn't request this, you don't need to do anything; you won't "
-    "receive any more email from us. Please do not reply to this email. "
-    "For more information, check our Help Center here: ")}
-
+${_("If you didn't create an account, you don't need to do anything; you "
+      "won't receive any more email from us. If you need assistance, please "
+      "do not reply to this email message. Check the help section of the "
+      "{platform_name} website.").format(platform_name=configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME))}
   ${settings.ZENDESK_URL}
```
Comment (stvstnfrd) [inline lms/templates/emails/activation_email.txt]: @caesar2164 We should consider linking to `marketing_link('FAQ')`

Comment (caesar2164) [reply lms/templates/emails/activation_email.txt]: yes we should! Wanna just do it?

**Pull request merged**